reactive-streams/src/main/scala/fs2/interop/reactivestreams/StreamSubscriber.scala

```diff
@@ -69,15 +69,19 @@ final class StreamSubscriber[F[_], A](
     sub.onError(t)
   }
 
-  def stream(subscribe: F[Unit]): Stream[F, A] = sub.stream(subscribe)
+  def streamChunk(subscribe: F[Unit]): Stream[F, Chunk[A]] = sub.stream(subscribe)
+  def stream(subscribe: F[Unit]): Stream[F, A] = streamChunk(subscribe).unchunks
 
   private def nonNull[B](b: B): Unit = if (b == null) throw new NullPointerException()
 }
 
 object StreamSubscriber {
 
+  def apply[F[_]: Async, A](bufferSize: Long): F[StreamSubscriber[F, A]] =
```

Comment:
The `bufferSize` should be an `Int` here as `Chunk` is `Int` based.

Reply:
https://github.com/typelevel/fs2/pull/2666/commits/f112b4ccb8da9ac7f2c1dae413376cc6bd150ecc